.ci_support/linux_64_numpy1.16python3.6.____cpython.yaml

```diff
@@ -28,6 +28,10 @@ jasper:
 - '1'
 jpeg:
 - '9'
+libcblas:
```

Comment:
This actually might be an important change.

We likely didn't rerender after this got added
https://github.com/conda-forge/opencv-feedstock/pull/255/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR90